backend/corpora/common/entities/dataset.py

```diff
@@ -0,0 +1,28 @@
+import typing
+
+from .entity import Entity
+from ..corpora_orm import DbDataset
+
+
+class Dataset(Entity):
+    tables = [
+        DbDataset,
+    ]
+
+    def __init__(self, db_obj: DbDataset):
```

Comment:
I'm not sure that the Dataset entity class should be initialized with a DbDataset. The primary use case of this class is to be able to take in the input from a user who is inputting a dataset and parse it into a Dataset object that can be transformed into a DbDataset object to be placed into the database. For this, we actually do need all the attributes available to set similar to the function signature in project.py.

Reply:
In that case I think we should create a function call `new` or `create` that takes in all of those parameters, and  generates a DBDataset to pass into the `__init__`. It would be similar to get, except no querying.

Reply:
These details can be fleshed out when we add save functionality.